compiler/luci/service/src/Nodes/CircleFullyConnected.test.cpp

```diff
@@ -59,3 +61,111 @@ TEST(CloneNodeTest, clone_FullyConnected_wf_NEG)
   auto cloned = luci::clone_node(node_fc, gc.get());
   ASSERT_EQ(nullptr, cloned);
 }
+
```

Comment:
How about adding some static cass such as `fully_connected_static_shape`?

Reply:
I saw your comment just now. I will also add a case for static dimensions.
However, if we classify it in the same way as the dynamic case, it may be difficult to generate a negative case.😢

Reply:
Oh, I see.
You don't necessarily have to add that test, so don't worry too much about it.
😅